Solutions/CohesitySecurity/Analytic Rules/RansomwareAlert.yaml

```diff
@@ -0,0 +1,76 @@
+id: 52a51512-5c38-4e0e-a741-aa2afe6c8d34
+name: Ransomware Alert
+description: Take custom fields in alert
+severity: High
+requiredDataConnectors: []
+queryFrequency: 5m
+queryPeriod: 5m
+triggerOperator: gt
+triggerThreshold: 0
+tactics: []
+relevantTechniques: []
+query: |
+  let outliers =
+      SecurityAlert
+      | where TimeGenerated > now() - 300d
```

Comment:
suggest setting this at the maximum lookback to 14d for perf reeasons.

Reply:
just update this.